README.md

```diff
@@ -87,6 +87,22 @@ Show message with ID 1
 
     RETR 1
 
+## Use with docker-machine
+
+Because service clients like the addon-administration who are using the
```

Comment:
I'm not sure, but I think there is a comma necessary: "service clients, like the addon-administration"